### PR TITLE
Add token image fallback

### LINKED ISSUE #157: When new token is added, it shows missing image
image

### PR DESCRIPTION
**Pull request by jubalm**
Resolves #157 

Tokens that don't have icons on the website library will show this icon as a fallback

Screenshot 2023-07-06 at 3 02 48 PM

### CODE REVIEW

File: app/ts/components/TokenManager/index.tsx [hunk 1]
```diff
@@ -113,7 +113,9 @@ const TokenCard = ({ token, onSelect }: TokenCardProps) => {
 	return (
 		<div class='aspect-square bg-neutral-800 p-4 flex items-end relative group outline-none' tabIndex={1}>
 			<div class='relative'>
-				<img class='w-12 h-12 mb-2' src={`/img/${token.address.toLowerCase()}.svg`} />
+				<object class='w-12 h-12 mb-2' data={`/img/${token.address.toLowerCase()}.svg`} type="image/png">
+					<img class='w-12 h-12' src="/img/ethereum.svg" />
+				</object>
```
Comment (MicahZoltu) [inline app/ts/components/TokenManager/index.tsx:118]: How does this result in a fallback image?

File: app/ts/components/TokenManager/index.tsx [hunk 2]
```diff
@@ -113,7 +113,9 @@ const TokenCard = ({ token, onSelect }: TokenCardProps) => {
 	return (
 		<div class='aspect-square bg-neutral-800 p-4 flex items-end relative group outline-none' tabIndex={1}>
 			<div class='relative'>
-				<img class='w-12 h-12 mb-2' src={`/img/${token.address.toLowerCase()}.svg`} />
+				<object class='w-12 h-12 mb-2' data={`/img/${token.address.toLowerCase()}.svg`} type="image/png">
+					<img class='w-12 h-12' src="/img/ethereum.svg" />
```
Comment (MicahZoltu) [inline app/ts/components/TokenManager/index.tsx:117]: We shouldn't use the ETH logo as a fallback for missing logo.  Consider a question mark or some other indicator of "unknown" to clearly indicate that we don't have an icon.

**Review by MicahZoltu — CHANGES_REQUESTED** [review]
I would like to see something other than the ETH logo as a fallback before this is merged.

File: app/ts/components/TokenManager/index.tsx (hunk 1 above)
Comment (jubalm) [reply app/ts/components/TokenManager/index.tsx:118]: an object element can load an external resource, just like how a video behaves, and will fallback to the underlying `` while it fetches or if it failed to fetch the resource.

File: app/ts/components/TokenManager/index.tsx (hunk 2 above)
Comment (jubalm) [reply app/ts/components/TokenManager/index.tsx]: Replaced the icon with this:
Screenshot 2023-07-06 at 3 02 48 PM

Comment (MicahZoltu) [reply app/ts/components/TokenManager/index.tsx]: Hmm, screenshot missing?

**Review by MicahZoltu — APPROVED** [review]

**Pull request merged**